cypress/integration/mytools.js

```diff
@@ -5,6 +5,34 @@ describe('Dockstore my tools', function() {
      cy.visit(String(global.baseUrl) + "/my-tools")
   });
 
+  describe('Should contain extended DockstoreTool properties', function() {
+    it('visit another page then come back', function() {
+      cy.get('a').contains('Tools').click()
```

Comment:
Let's try IDs here too, and maybe use Documentation or the Accounts page because Tools or Workflows might not be long for this world